package.json

```diff
@@ -33,24 +33,19 @@
     "client library"
   ],
   "dependencies": {
-    "async": "^2.0.0",
-    "gtoken": "^1.1.0",
-    "lodash.noop": "^3.0.0",
-    "jws": "^3.1.0",
-    "request": "^2.74.0",
-    "string-template": "^1.0.0"
+    "gtoken": "^1.2.1",
+    "lodash.noop": "^3.0.1",
+    "jws": "^3.1.4",
+    "request": "^2.79.0"
```

Comment:
We've a couple options:

* I change the `request` version back to `^2.74.0` right away
* We leave it at `^2.79.0` for now, and trust our 0.10.0 tests to catch any future incompatibilities that might require us to downgrade or lock to a specific version
* We drop support for Node.js 0.10 and 0.12, making this upgrade to `^2.79.0` okay